example/index.html

```diff
@@ -1,23 +1,26 @@
 ---
-layout: default
+layout: page
 ---
 
 <div class="home">
 
   <h1 class="page-heading">Posts</h1>
 
+  {% assign custom_url = site.url | append: site.baseurl %}
+  {% assign full_base_url = site.github.url | default: custom_url %}
```

Comment:
If the user sets a base url, wouldn't this result in `http://username.github.io/project/project/`?